agent/bench-scripts/test-bin/mock-cmd

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
 
 echo "$0 $*" >> $_testlog
+
+prog=$(basename "$0")
+last=$(echo $* | sed 's/.* //')
```

Comment:
Can we do something like the following to avoid using `sed` (causing an extra fork):
```
args=( "$@" )
lastidx=$((${#args[@]} - 1))
last=${args[$lastidx]}
```